content/_layouts/frame.html.haml

```diff
@@ -17,12 +17,12 @@
     %meta{:charset => "utf-8"}/
     %meta{:content => "width=device-width, initial-scale=1", :name => "viewport"}/
     %meta{:content => "ie=edge", "http-equiv" => "x-ua-compatible"}/
-    %link{:href => expand_link("img/favicon.ico"), :rel => "icon", :sizes => "32x32", :type => "img/png"}/
-    %link{:href => expand_link("img/favicon-32x32.png"), :rel => "icon", :sizes => "32x32", :type => "image/png"}/
-    %link{:href => expand_link("img/favicon-16x16.png"), :rel => "icon", :sizes => "16x16", :type => "image/png"}/
-    %link{:href => expand_link("img/apple-touch-icon-76x76.png"), :rel => "apple-touch-icon", :sizes => "76x76"}/
-    %link{:href => expand_link("img/apple-touch-icon-120x120.png"), :rel => "apple-touch-icon", :sizes => "120x120"}/
-    %link{:href => expand_link("img/apple-touch-icon-152x152.png"), :rel => "apple-touch-icon", :sizes => "152x152"}/
+    %link{:href => expand_link("sites/default/files/images/favicon.ico"), :rel => "icon", :sizes => "32x32", :type => "images/png"}/
```

Comment:
Since this is an `ico` file, should be `image/x-icon` (per https://stackoverflow.com/a/9943834/3043709 ) 
```suggestion
    %link{:href => expand_link("sites/default/files/images/favicon.ico"), :rel => "icon", :sizes => "32x32", :type => "image/x-icon"}/
```